robocop/run.py

```diff
@@ -23,17 +26,25 @@ def __init__(self):
         self.load_reports()
 
     def set_output(self):
+        """ Set output for printing to file if configured. Else use standard output """
         self.out = self.config.output or sys.stdout
 
     def write_line(self, line):
+        """ Print line using file=self.out parameter (set in `set_output` method) """
         print(line, file=self.out)
 
     def run(self):
+        """ Entry point for running scans """
         self.recognize_file_types()
         self.run_checks()
         self.make_reports()
 
     def recognize_file_types(self):
+        """
+        Preparse files to recognize types. If the filename is "__init__.robot" file type is INIT.
+        If the file is imported somewhere then file type is RESOURCE. Otherwise file type is GENERAL.
+        Those types are important since are used to define parsing class for robot API.
```

Comment:
I would rephrase it to:
```suggestion
        Prepare files to recognize their types. If the filename is `__init__.robot`, the type is `INIT`.
        If the file is imported somewhere then file type is `RESOURCE`. Otherwise file type is `GENERAL`.
        These types are important since they are used to define parsing class for robot API.
```

Reply:
it's actually "preparse" as we're parsing files (in limited way) to detect file type. So what could be best name?

Reply:
Oh, ok 😄 I updated the suggestion.